serializer-deserializer/pom.xml

```diff
@@ -221,6 +232,12 @@
             <groupId>com.google.protobuf</groupId>
             <artifactId>protobuf-java</artifactId>
         </dependency>
+        <!-- Temporarily adding dependency on a transitive dependency to fix security bug in underlying library. Remove when dependency is upgraded. -->
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-collections4</artifactId>
```

Comment:
Do we need to remove the old collections dependency? Or is it a transitive dependency?

Reply:
it is a transitive dependency, I have excluded it